omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -441,6 +471,9 @@
 ####
 default['private_chef']['oc_chef_authz']['http_init_count'] = 25
 default['private_chef']['oc_chef_authz']['http_max_count'] = 100
+# The queue max is only effective if authz_pooler_timeout (in the
```

Comment:
Does `authz_pooler_timeout` actually work like this? I'm going to go into the `opscoderl_httpc` library to check that out. I'm a bit surprised that we're not surfacing more tunables here and that they're not exposed via the main `pooler` config.